### PR TITLE
Removes "Special Verbs" category & reorganises admin buttons categories

### PR DESCRIPTION
**Pull request by peoplearestrange**
## About The Pull Request

I've re-organised all the admin buttons into 3 new categories and kept 2 of the old ones
NEW:
Administration = All "paperwork" style tools and base commands for players
Admin - Game = All investigative tools and admin tools that effect the game
Admin - Events = All tools used to make/manage/build on IC concepts
SAME:
Fun = Game Breaking Meme's we all love
Server = ACTUAL server effecting commands, ones that effect OOC and server stuff

I've also moved a few outliers to OOC (Link Discord, Notify restart) and debug (delete and check contents). As these are either used by players or used only for debugging really

I also completely removed the "Hide Most" button verb as its now simply not needed given most buttons have spread out and its less of a cluster cuss. I left the "hide all" as it seems to squash it down to the bare min nicely. This is now in the "Administration tab"

Here is a breakdown of all the movements:
https://docs.google.com/spreadsheets/d/1JAwY93VkfdcquTJqkFEu2hO5TB0jF6ydKKnTXPTSvZg/edit?usp=sharing

## Why It's Good For The Game

I realise this is going to upset admins at first, my god do we all hate change. But the organisation is much more logical and removes the god awful "Special Verbs" tab which honestly was a bastard of left over bits.

It also allows for more obviously labelled categories in the tabs, and thus reducing buttons in each tab.

## Changelog
:cl:
del: Removed Special Verbs Tab
admin: New admin button Tabs
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
based change

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -296,20 +296,11 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 		/client/proc/readmin
 		)
 
-/client/proc/hide_most_verbs()//Allows you to keep some functionality while hiding some verbs
```
Comment (optimumtact) [inline code/modules/admin/admin_verbs.dm]: why did you remove this

Comment (peoplearestrange) [reply code/modules/admin/admin_verbs.dm]: Apparently because Im over tired... Ill add it back

Comment (optimumtact) [reply code/modules/admin/admin_verbs.dm]: tbh looks pretty useless

Comment (peoplearestrange) [reply code/modules/admin/admin_verbs.dm]: Oh now I see, I removed too much, One extra line got caught in the delete of the "Hide most"

**Comment by kriskog** [comment]
Getting used to new positions is going to be a pain, but it needed to be done. :+1:

**Comment by Dorsisdwarf** [comment]
if an admin is upset at the verb tornado being tamed I will destroy their very soul 👍

**Comment by cacogen** [comment]
you need some of this to debug sometimes and with bad eyes hopefully this will be better

**Review by Cyberboss — DISMISSED** [review]

**Comment by Cyberboss** [comment]
God given PR

**Review by peoplearestrange — COMMENTED** [review]
Made the changes

**Comment by PKPenguin321** [comment]
PSA to admins, if you use tabs instead of the command line you're doing it wrong

**Comment by peoplearestrange** [comment]
> PSA to admins, if you use tabs instead of the command line you're doing it wrong

True but its a helluva lot easier to teach people buttons when learning. Plus not everything is labelled consistently so remembering the right verb isn't always that simple

**Comment by PKPenguin321** [comment]
>remembering the right verb isn't always that simple

learn to use the autofill to search for verbs faster, trust me it's super worth it

**Comment by peoplearestrange** [comment]
Ergggg, because of the way it sorts things Administration is now last in the tabs. Like this:
image

I feel like, this might be an issue. Will probably make "Administration" back into "Admin" to avoid this.

**Comment by peoplearestrange** [comment]
Changes made, I think thats it. Will download and test it

**Review by Cyberboss — APPROVED** [review]

**Comment by itseasytosee** [comment]
Why is start game not in server and where is it now.

**Comment by cacogen** [comment]
Start Now is under Admin - Events. Should be in Server with things like End
Round, Reboot World, Change Map, etc.(

**Comment by peoplearestrange** [comment]
Its in events because thats when its usually used. When its paused for an event and then resumed.
If it annoys people I could move it to server

**Comment by Cyberboss** [comment]
#49216
